pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -38,105 +30,147 @@ parts = rf.select(
 )
 
 print("CRS", crs.value.crsProj4)
-parts.show(5)
+parts.show(5, False)
 
 ```
 
-Let's select a single tile and view it.
+You can also see that the single raster has been broken out into many arbitrary non-overlapping regions. Doing so takes advantage of parallel in-memory reads from the cloud hosted data source and allows Spark to work on managable amounts of data per task. 
 
-```python
-tile = rf.select(rf_tile("proj_raster")).first()[0]
-display(tile)
-# todo verify its chilly
+```python count_by_uri
+rf.groupby(rf.proj_raster_path).count().show()
 ```
 
+Let's select a single tile and view it. The tile preview image as well as the string representation provide some basic information about the tile: it's dimensions as number of columns and rows, and the cell type, or data type of all the cells in the tile. For more about cell types, refer to @ref:[this discussion](nodata-handling.md#cell-types).
 
-### Single Scene, Multiple Bands
+```python show_tile_sample
+tile = rf.select(rf_tile("proj_raster")).first()[0]
+display(tile)
+```
 
-Generally we'll want to work with multiple bands. If you have several singleband image files of the same scene, `raster` can make use of a CSV (or other DataFrame) specifying the band columns and scene rows.
+## URI Formats
 
-```python
-csv = """
-red,green,blue
-{},{},{}
-""".format(s2_band(4), s2_band(3), s2_band(2))
+/[ TODO Populate this with valid URI's schemes we can use with an example? including how to use gdal /]
 
-print(csv)
+file://  (is this only valid for spark local? or if file is )
+https://
+http:// ?
+s3a://
+s3n:// ?
+hdfs:// ?
+ftp:// ?
+azure ?
+google cloud ?
```

Comment:
We will need to test anything we say we support.

Reply:
perhaps if u can create a unit test I can write it up saying here is what we know works. Let me know